### ISSUE TITLE
Warning level was reported on error level issues

### ISSUE THREAD

**Issue by siralam** [irgaly/danger-sarif#2]
Hi @irgaly , thanks for the help on the sample PR conversation!

After I got it to work, I found one thing though - in my sarif file there is a `error` level issue, but it seems danger still treat it as a `warning`, and therefore Github Action bot also uses the warning format to comment on the PR.

I am not sure if this is something the plugin can control though

Below is the error level issue in the `.sarif` file (I am removed some sensitive information)

```
"results": [
                {
                    "level": "error",
                    "locations": [
                        {
                            "physicalLocation": {
                                "artifactLocation": {
                                    "uri": "../../../com/example/android/core/network/adapter/BigDecimalAdapter.kt",
                                    "uriBaseId": "%SRCROOT%"
                                },
                                "region": {
                                    "endColumn": 88,
                                    "endLine": 11,
                                    "startColumn": 73,
                                    "startLine": 11
                                }
                            }
                        }
                    ],
                    "message": {
                        "text": "The method `java.math.BigDecimal.toPlainString` has been forbidden: BigDecimal.toPlainString() ignores locale number formatting. Use formalDecimalPlaces() instead."
                    },
                    "ruleId": "detekt.style.ForbiddenMethodCall"
                }
            ]
```

And danger generates the below:
![image](https://github.com/irgaly/danger-sarif/assets/28033467/b8762221-4d9f-4c03-a268-ff93014902b5)

**Comment by irgaly** [comment]
@siralam 

Thank you for your reporting this issue!
I think danger-sarif should treat `error` level as `Danger.fail()`, and let CI failed if SARIF has error results.

I implemented this behavior by #3, then release danger-sarif [v0.9.1](https://github.com/irgaly/danger-sarif/releases/tag/v0.9.1). Please try new version.

**Comment by siralam** [comment]
@irgaly Really thanks for working so promptly 🙇‍♂️  0.9.1 worked!
